requirements.txt

```diff
@@ -0,0 +1,207 @@
+absl-py==2.1.0
```

Comment:
i'd say that we can remove this for now because we will set basic requirements on the pip installation. :thinking:

Reply:
this = requirements.txt